### PR TITLE
[DGR-682] - Update Apiary doc

### PR DESCRIPTION
**Pull request by shclarke007**
[DGR-682](https://accredible.atlassian.net/browse/DGR-682)
Screenshot 2024-09-04 at 13 18 31

Screenshot 2024-09-04 at 13 18 35

[DGR-682]: https://accredible.atlassian.net/browse/DGR-682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -428,6 +428,141 @@ Create multiple Credentials using this action. It requires a JSON object contain
...
+                        "group_name": "Test Group",
+                        "url": "https://www.example.com/10000005",
+                        "grade": null,
+                        "complete": true,
+                        "expired_on": null,
+                        "custom_attibutes": null,
+                        "course_link": null,
+                        "meta_data": null
+                    },
+                    "warnings": {
```
Comment (abhisheksarka) [inline apiary.apib:559]: This would be `error`

**Review by abhisheksarka — COMMENTED** [review]
Are we waiting on the confirmation for the content? I don't see the text that is shared in the PR's description here

File: apiary.apib (hunk 1 above)
Comment (shclarke007) [reply apiary.apib:559]: oops. yes

**Comment by shclarke007** [comment]
> the content? I don't see t

Yes, I have sent to Kelly to confirm as nothing in the ticket so re-used some of v1 text

**Review by kei178 — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -428,6 +428,145 @@ Create multiple Credentials using this action. It requires a JSON object contain
...
+                        "course_link": null,
+                        "meta_data": null
+                    },
+                    "errors": {
+                        "code": "insufficient_credits",
+                        "message": "The credential was created but not published."
+                    }
+                }
+            ]
+        }
```
Comment (kei178) [inline apiary.apib:569]: Wouldn't it be better to also mention the only error result scenario?

**Review by albertpaulp — APPROVED** [review]

**Pull request merged**